modules/tooltip/react/stories/Tooltip.stories.mdx

```diff
@@ -0,0 +1,84 @@
+
+import {Tooltip} from '@workday/canvas-kit-react-tooltip';
+
+import {Default} from './examples/Default';
+import {CustomContent} from './examples/CustomContent';
+import {DescribeType} from './examples/DescribeType';
+import {Placements} from './examples/Placements';
+import {UseTooltip} from './examples/UseTooltip';
+
+<Meta title="Components/Popups/Tooltip/React" component={Tooltip} />
+
+# Canvas Kit React Tooltips
+
+A Tooltip component that renders information/text when the user hovers over an element. A tooltip is
+used to label or describe an element. By default, a tooltip will label an element. This is useful
+for IconButtons. A tooltip can also be used to describe additional information about an element
+
+[> Workday Design Reference](https://design.workday.com/components/popups/tooltips)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-react-tooltip
+```
+
+## Usage
+
+This component follows the
+[W3 Tooltip specification](https://www.w3.org/TR/wai-aria-practices/#tooltip). Tooltips are used to
+label icon buttons and provide additional context to elements.
+
+### Basic Example
+
+Here is a basic example of an `IconButton` using a tooltip to label the icon. This is labeling the
+button for both sighted users and for screen readers. A tooltip will provide an `aria-label` to
+child elements for the accessibility tree and a visual tooltip during mouse hover and focus events.
+Mouse over or focus on the x icon below. Click "Show code" to see the source code.
+
+<ExampleCodeBlock code={Default} />
+
+### Describing an Element
+
+The default mode for a tooltip is to label content via `aria-label`. If a tooltip is meant to
+provide ancilary, but not critical information, the `type` can be set to `describe`. This will add
+`aria-describedby` to the target element.
+
+<ExampleCodeBlock code={DescribeType} />
+
+### Custom CustomContent
+
+A tooltip can contain HTML, but should not contain any focusable elements or semantically meaningful
+formatting. The content will loose all semantic meaning when read by a screen reader. If complex
+content or a focusable element is needed by your UI, a tooltip is not a good choice. Consider using
+a dialog instead.
+
+<ExampleCodeBlock code={CustomContent} />
+
+### Placements
+
+The tooltip allows for a `placement` configuration. The tooltip uses
+[PopperJS](https://popper.js.org/) to position tooltips, so any valid PopperJS placement is valid
+for tooltips.
+
+<ExampleCodeBlock code={Placements} />
+
+### The UseTooltip Hook
+
+The `Tooltip` component is a combination of the `TooltipContainer` (a styled element), `Popper`
+(which uses PopperJS and the popup stack), the `useTooltip` hook and some behavior. If custom
+behavior is required, these sub-components can be composed in a custom container element. This
+example uses those parts directly while being functionally equivelent to the original basic example.
```

Comment:
equivalent

Reply:
You can use the suggestion feature of Github:

![image](https://user-images.githubusercontent.com/338257/109227535-b5399800-777d-11eb-887b-807d5846466e.png)

It appears like this:
```suggestion
example uses those parts directly while being functionally equivalent to the original basic example.
```